### PR TITLE
Functionalandclasscomponents

### LINKED ISSUE #1256: Functional Components and Class Components in React
Description 📜
What is Functional Components and Class Components in ReactJS

The domain of Contribution 📊 will be Document, Code.

I am a GWOC'21 Contributor from Mern 3 Batch.
@Salonee-Jain @manjeetdhayal  Please /assign this issue to me.

### PR DESCRIPTION
**Pull request by Drontitan**
<hr>

## Description 📜

I have created the md file containing information about Functional and class based components
Fixes #1256 

<hr>

## Type of change 📝

- [ ] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [ ] Frontend Dev React/Angular/Vue

<hr>

## Checklist ✅

- [x ] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x ] I have performed a self-review of my own code or work.
- [x ] I have commented my code, particularly in hard-to-understand areas.
- [x ] My changes generates no new warnings.
- [ x] I'm GWOC'21 contributor

<hr>

### CODE REVIEW

File: Frontend_Web_Development_React_Angular_Vue/React/Week 2/Functionalandclasscomponents.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+```javascript
+function withdraw(account, amount) {
+  account.total -= amount;
+}
+```
+
+*React is pretty flexible but it has a single strict rule:*
+
+**All React components must act like pure functions with respect to their props.**
+
```
Comment (heet-kumar) [inline Frontend_Web_Development_React_Angular_Vue/React/Week 2/Functionalandclasscomponents.md:228]: Try to add reference link at the bottom from where you have referred and created this PR.

**Review by heet-kumar — CHANGES_REQUESTED** [review]
add this section at the bottom

**Comment by heet-kumar** [comment]
@Drontitan this issue is from MERN Domain so you need to send this in MERN channel in Discord you have posted this issue in Frontend with Framework.

**Comment by heet-kumar** [comment]
@anandhu720 this issue #1256 come under MERN Domain this PR is generated for the MERN domain.
already in our Domain one PR is already merged on this particular topic.

**Comment by Drontitan** [comment]
@anandhu720 pls check the pr now i have made the changes what @heet-kumar have said

**Comment by anandhu720** [comment]
@Drontitan bro this issue was created for MERN branch i think so.Because this issue is already been created and merged in our branch can you please cross check it

**Comment by Drontitan** [comment]
@anandhu720  means now this pr is wasted so i have to take new 😶

**Comment by Drontitan** [comment]
can i close this pr

**Comment by anandhu720** [comment]
sure.please check milestone 3 for our issues

**Pull request closed without merging**